app/controllers/api/v1/schools_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::SchoolsController < ApplicationController
+
+  # GET /schools
+  def index
+    @schools = School.all
+    render json: @schools
```

Comment:
This works but doesn't give you control over the API being different than the internal data structure, which means you can't change the data structure without possibly breaking the API.  What I'd recommend is a layer between the ActiveRecord object and the JSON that is produced.  That way you can also make a v2 API that is different from the v1 API without them both being tied directly to the same model.  

Because we like our systems to have API documentation via Swagger, we've solved the above issue on other applications by making Swagger bindings that are this other layer.

Take a look through 
* [Highlights README about swagger](https://github.com/openstax/highlights-api#swagger-clients-and-bindings)
* [Highlights swagger definitions for models](https://github.com/openstax/highlights-api/blob/master/app/controllers/api/v0/highlights_swagger.rb)
* [where swagger document is defined and made available through an endpoint](https://github.com/openstax/highlights-api/blob/master/app/controllers/api/v0/swagger_controller.rb)
* [an example use to read JSON input](https://github.com/openstax/highlights-api/blob/master/app/controllers/api/v0/highlights_controller.rb#L10)
* [an example of using a binding to convert a model to JSON](https://github.com/openstax/highlights-api/blob/master/app/controllers/api/v0/highlights_controller.rb#L18-L19)
* [an example of an autogenerated binding](https://github.com/openstax/highlights-api/blob/master/app/bindings/api/v0/bindings/new_highlight.rb)

and then let's talk